docker/python-plotting/1.0.0/plot_umap.py

```diff
@@ -0,0 +1,13 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import sys
+
+# Read UMAP embedding
+umap = pd.read_csv(sys.argv[1], index_col=0)
+
+# Plot UMAP
+plt.scatter(umap["UMAP1"], umap["UMAP2"])
+plt.xlabel("UMAP1")
+plt.ylabel("UMAP2")
+plt.title("UMAP Embedding")
+plt.savefig("umap.png")  
```

Comment:
Small nit: can we take 2 args where the second is the output file name? And at that point it probably makes sense to use `argparse`